File: FitToFit/UI/fittofit_admin/lib/models/treninziUpdateRequest.dart
class TreninziUpdateRequest {
  String opis;
  int maxBrojClanova;
  double cijenaPoTerminu;
  String trajanje;
  double prosjecnaPotrosnjaKalorija;
  int vrstaId;
  String namjena;
  String? slika;

  TreninziUpdateRequest(
      {required this.opis,
      required this.maxBrojClanova,
      required this.cijenaPoTerminu,
      required this.trajanje,
      required this.prosjecnaPotrosnjaKalorija,
      required this.vrstaId,
      required this.namjena,
      this.slika});

  Map<String, dynamic> toJson() {
    return {
      'opis': opis,
      'maxBrojClanova': maxBrojClanova,
      'cijenaPoTerminu': cijenaPoTerminu,
      'trajanje': trajanje,
      'prosjecnaPotrosnjaKalorija': prosjecnaPotrosnjaKalorija,
      'vrstaId': vrstaId,
      'namjena': namjena,
      'slika': slika,
    };
  }
}

File: FitToFit/FitToFit.Model/Requests/TreninziUpdateRequest.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FitToFit.Model.Requests
{
    public class TreninziUpdateRequest
    {
        public string Opis { get; set; } = null!;

        public int MaxBrojClanova { get; set; }

        public decimal CijenaPoTerminu { get; set; }

        public string Trajanje { get; set; } = null!;

        public decimal ProsjecnaPotrosnjaKalorija { get; set; }

        public int VrstaId { get; set; }

        public string Namjena { get; set; } = null!;

        public byte[]? Slika { get; set; }
    }
}
